omop_examples/extract_drug_treatments_from_a_molecule.ipynb

```diff
@@ -0,0 +1,278 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d4753f46-ea81-471e-9cfb-fdab151e6267",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# CMS SynPuf: How to extract drug treatments containing a specific molecule\n",
+    "\n",
+    "This notebook queries the [CMS SynPuf dataset](https://console.cloud.google.com/marketplace/product/hhs/synpuf?pli=1), a public synthetic patient data in OMOP. This notebook is intended to be used as an example for how to query the public OMOP dataset, and how to do basic visualizations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f8336bd-793c-4cba-9217-0927e1b16f87",
+   "metadata": {},
+   "source": [
+    "## Import python libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "71671891-b357-4b3d-9af1-1b5a67a1f340",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from google.cloud import bigquery\n",
+    "\n",
+    "# Enable IPython to display matplotlib graphs.\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab938992-40b0-4fee-976a-150665bb0903",
+   "metadata": {},
+   "source": [
+    "## Notebook setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3725ee38-87f0-408d-bb14-6a027ab7409f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "'''\n",
+    "Resolves bucket URL from bucket reference in workspace.\n",
+    "'''\n",
+    "def get_bucket_url_from_reference(bucket_reference):\n",
```

Comment:
I don't think you need this function any more?

Reply:
Yes you're right I don't need them, thanks for catching. They're removed.

What I had forgot to do was to use the BQ dataset reference instead of the hard coded string. This is fixed now.

Reply:
(also, IIRC the orig HHR nbs also have that semantic mismatch where they call it a bucket, but it's a BQ ref... I'd mentioned that to them but I think it got dropped)